controllers/articles.controller.js

```diff
@@ -0,0 +1,18 @@
+const { selectArticlesById } = require("../models/articles.model")
+
+exports.getArticlesById = (request, response, next) => {
+
+   const { article_id } = request.params
+
+    selectArticlesById(article_id)
+    .then((article) => {
+        if (!article)
+            return next({status: 404, msg: "Article not found"})
```

Comment:
You have handled this on line 9 of your model, and the message there lines up with your tests. I'd suggest removing this line in the controller